app/api.py

```diff
@@ -1,10 +1,21 @@
+from app import db
 from flask import jsonify
+from flask import request
+from .models import Payment
 from flask.views import MethodView
 
 
 class PaymentResource(MethodView):
-    def get(self):
-        return jsonify(ok=True)
-
     def post(self):
-        pass
+        content = request.json
```

Comment:
`request.json` returns `None` if no JSON was send. We need to check it or bellow line will fail with `TypeError`